.gitignore

```diff
@@ -1,3 +1,6 @@
 /target/
 Cargo.lock
 *~
+**/*.rs.bk
+[._]*.sw?
+[._]sw?
```

Comment:
I'm curious, what kind of files are these? Vim swap files? What is the initial `+` about?

Reply:
- `*.rs.bk*` - `rustfmt` backup files, in case if you want to `diff` the original and the result.
- `.*.sw?` - vim swap files that match `.*.swp`, `.*.swo`, `.*.swn` etc. (the last letter runs backwards) and `_*.sw?` for its Windows counterpart
- `.sw?` - vim swap files for directories (just `.swp`, `.swo` etc., no filename part), also matches those for Windows

Reply:
Great, thanks for the info!

Reply:
Actually, the (presumably complete) set of possible swap file names is even more complex: https://unix.stackexchange.com/a/326737. I just don't bother with this in my projects, though I don't know if that can be applied to yours'.